src/Native/System.Native/pal_interfaceaddresses.cpp

```diff
@@ -44,8 +44,7 @@ extern "C" int32_t EnumerateInterfaceAddresses(IPv4AddressFound onIpv4Found,
         {
             // IP Address
             IpAddressInfo iai = {
-                .InterfaceIndex = interfaceIndex,
-                .NumAddressBytes = NUM_BYTES_IN_IPV4_ADDRESS,
+                .InterfaceIndex = interfaceIndex, .NumAddressBytes = NUM_BYTES_IN_IPV4_ADDRESS,
```

Comment:
Last time this was brought up, I think everyone agreed that it looks better on multiple lines; however, I couldn't find the right option to do that. Also, any changes to the formatting style guidelines would be in a different PR :)